deploy/ansible/roles/legion_enclave/templates/event-collector-values.yaml.j2

```diff
@@ -0,0 +1,16 @@
+rootDomain: {{ root_domain }}
+enclaveName: "{{ enclave }}"
+domainDelimiter: "{{ domain_delimiter }}"
+
+replicas: "{{ collector.replicas }}"
+
+image:
+  repository: "{{ nexus_docker_repo }}/"
+  tag: "{{ base_version }}-{{ local_version }}"
+
+s3:
```

Comment:
Do we still need given we have kube2iam?

Reply:
@alexander-semenets same question

Reply:
@akharlamov kube2iam-branch is not ready yet